book/content/rdm/docmetadata.md

```diff
@@ -1,19 +1,15 @@
 # Documentation and metadata
 
-Having data available is of no use if it cannot be understood. For example a table of numbers is useless if there are no
-headings which describe what the columns/rows contain. Therefore you should ensure that open datasets include consistent
-core metadata, and that the data is fully described. This requires that all documentation accompanying data is written
-in clear, plain language, and that data users have sufficient information to understand the source, strengths,
-weaknesses, and analytical limitations of the data so that they can make informed decisions when using it.
+Having data available is of no use if it cannot be understood. For example a table of numbers is useless if there are no headings which describe what the columns/rows contain.
+Therefore you should ensure that open datasets include consistent core metadata, and that the data is fully described.
+This requires that all documentation accompanying data is written in clear, plain language, and that data users have sufficient information to understand the source, strengths, weaknesses, and analytical limitations of the data so that they can make informed decisions when using it.
 
 - The level of documentation and metadata will vary according to the project and the range of people the data needs to
   be understood by
-- It is best practice to use recognised community metadata standards to make it easier for datasets to be combined. For
-  example, for brain data the [Brain Imaging Data Structure](https://doi.org/10.25504/FAIRsharing.rd1j6t) is the
-  standards to use; other [metadata standards](https://fairsharing.org/standards)(reporting requirements,
-  terminologies, and models/schemas) are searchable in FAIRsharing.
-- Variables should be defined and explained using
-  [data dictionaries](http://help.osf.io/m/bestpractices/l/618767-how-to-make-a-data-dictionary)
+- It is best practice to use recognised community metadata standards to make it easier for datasets to be combined.
+For example, for brain data the [Brain Imaging Data Structure](https://doi.org/10.25504/FAIRsharing.rd1j6t) is the standards to use; other [metadata standards](https://fairsharing.org/standards)(reporting requirements, terminologies, and models/schemas) are searchable in FAIRsharing.
+- Variables should be defined and explained using [data dictionaries](http://help.osf.io/m/bestpractices/l/618767-how-to-make-a-data-dictionary)
```

Comment:
I think this link is broken: 
"oops
The page you were looking for doesn't exist"

Reply:
Ah! great catch. The original author of this chapter had worked on this last December. I will remove this.